traceloop-sdk/tracing.go

```diff
@@ -33,22 +33,28 @@ func newTracerProvider(ctx context.Context, serviceName string, exp trace.SpanEx
 	)
 
 	if err != nil {
-		panic(err)
+		return nil, err
 	}
 
 	return trace.NewTracerProvider(
 		trace.WithBatcher(exp),
 		trace.WithResource(r),
-	)
+	), nil
 }
 
 func (instance *Traceloop) initTracer(ctx context.Context, serviceName string) error {
 	exp, err := newOtlpExporter(ctx, instance.config.BaseURL, instance.config.APIKey)
 	if err != nil {
-		panic(err)
+		fmt.Printf("Failed to create otlp exporter: %v\n", err)
+		return err
```

Comment:
libraries in go shouldnt write to stdout
```suggestion
		return fmt.Errorf("initialize otlp exporter: %w", err)
```